src/utils/viewer/viewer.cpp

```diff
@@ -950,6 +970,9 @@ void Viewer::drawEvent() {
                  ImGuiWindowFlags_NoDecoration | ImGuiWindowFlags_NoBackground |
                      ImGuiWindowFlags_AlwaysAutoResize);
     ImGui::SetWindowFontScale(1.5);
+    if (flyingCameraMode_) {
+      ImGui::Text("In flying Camera MODE. Press key '3' to EXIT.");
+    }
```

Comment:
This seems unrelated to the bulk of changes in this PR.

Reply:
Yeah, I found it was annoying (without such info) when debugging this PR. So try to fix it in this PR.